libcontainer/cgroups/fs2/fs2.go

```diff
@@ -189,57 +189,41 @@ func (m *manager) Set(container *configs.Config) error {
 	if err := m.getControllers(); err != nil {
 		return err
 	}
-	var errs []error
```

Comment:
why?

Reply:
I think if there is an error, we should return without try other controllers.

Do you mean we should let enabled controllers success when we update more than 1 controller in one command, for example:
```
runc update --cpuset-cpus 1 --memory 5m test
```
We should let memory limit set success?

Reply:
In v1, it doesn't retry any other subsystems when got an error in one subsystem.

Reply:
make sense.